project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
```

Comment:
Would it be better to just leave this one in code manager?  Do any other projects use it?